src/Console/CheckoutCommand.php

```diff
@@ -11,50 +14,203 @@
 
 class CheckoutCommand extends Command
 {
+    /**
+     * Regex patterns for github repositories that should be ignored.
+     * These are repositories that don't have relevant API.
+     */
+    private array $ignoreModulePatterns = [
+        '/^silverstripe\/developer-docs$/',
+        '/^silverstripe\/installer$/',
+        '/^silverstripe\/vendor-plugin$/',
+        '/^silverstripe\/recipe-.*/',
+        '/-theme$/',
```

Comment:
These are repositories which either have no PHP code at all, or their PHP code isn't really relevant.
The second part of that might be fairly subjective though.... feel free to recommend changes here if you think of any changes that are appropriate.